.config/samples-config-v1.json

```diff
@@ -0,0 +1,175 @@
+        ],
+        "technologies": [
+            "Custom Function",
+            "SSO",
+            "Graph"
+        ]
+    },
+    "samples": [
+        {
+            "id": "Excel-HelloWorld-TaskPane-JS",
+            "onboardDate": "2024-07-01",
+            "title": "Excel Hello World Task Pane Add-in",
+            "shortId": "Excel Hello World",
+            "shortDescription": "A simple Excel Task Pane Add-in using JavaScript.",
+            "fullDescription": "This sample demonstrates how to create a simple Excel Task Pane Add-in using JavaScript.",
+            "types": [
+                "Excel"
+            ],
+            "tags": [
+                "JS",
+                "Hello World",
+                "Excel"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-HelloWorld-TaskPane-JS",
+            "onboardDate": "2024-07-01",
+            "title": "Word Hello World Task Pane Add-in",
+            "shortId": "Word Hello World",
+            "shortDescription": "A simple Word Task Pane Add-in using JavaScript.",
+            "fullDescription": "This sample demonstrates how to create a simple Word Task Pane Add-in using JavaScript.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "JS",
+                "Hello World",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-Add-in-AIGC",
+            "onboardDate": "2024-03-15",
+            "title": "Create a Word AIGC Add-in",
+            "shortId": "Word AIGC",
+            "shortDescription": "Inserting content generated by AI or predefined into document.",
+            "fullDescription": "This Add-in shows capabilities to insert content generated by AI or predefined into document and customize the style of document.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "TS",
+                "AIGC",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-Add-in-WritingAssistant",
+            "onboardDate": "2024-03-06",
+            "title": "Word Writing Assistant",
+            "shortId": "Word Writing Assistant",
+            "shortDescription": "A Word Add-in to assist writing with word document.",
+            "fullDescription": "This add-in demonstrates Word add-in capabilities to help users check errors, improve writing, and rephrase content.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "TS",
+                "Writing Assistant",
```

Comment:
I'm not sure what the value of tags that repeat the title are? I don't think we're likely to have multiple samples tagged with "Writing Assistant", are we?

Reply:
I don't the specific plan of adding new samples... But I think you guys can review and refine these tags or other properties in the config in a more thoughtful way once the migration is done.

Reply:
Btw, this tags are useful for matching purpose when users try to search specific sample.

Reply:
I would argue that it's the job of the product team to define how these tags are intended to be used, since they'll only be in the dev kit tool.

Reply:
I agree removing "writing assistant" as it's repeated with title.
For tags, we should align with the filterOptions to keep it as simple, unless we get new requirements. So we have 3 types: Platform(WXPO), Language(TS/JS), Technology - Technology could be tricky, but I think we can start with common topics among developers such as CF, SSO, NAA etc.
I am also thinking about a tag of sample skill level, like beginner/intermediate/advanced, but maybe we postpone that conversation till we have more candidates.